tauri-api/src/path.rs

```diff
@@ -193,7 +193,11 @@ fn app_name() -> crate::Result<String> {
     .expect("failed to get exe filename")
     .to_string_lossy();
 
-  Ok(app_name.to_string())
+  if cfg!(target_os = "windows") {
```

Comment:
```suggestion
#[cfg(target_os = "windows")]
```
How about this and `#[cfg(not(target_os = "windows"))]` below.
Conditional compilation can include the only code required. This could reduce the branch condition in actual code base.

Reply:
I tried that but the compiler kept yelling about missing semicolon. Anyway Lucas suggested we use `file_stem` so we won't need any conditions.